build.gradle

```diff
@@ -133,11 +134,32 @@ jacocoTestReport {
 
 project.tasks['sonarqube'].dependsOn jacocoTestReport
 
+pitest {
+  targetClasses = ['uk.gov.hmcts.reform.pcqbackend.*']
+  excludedClasses = [
+    'uk.gov.hmcts.reform.pcqbackend.configuration.*',
+    'uk.gov.hmcts.reform.pcqbackend.controller.*',
```

Comment:
Any reason why controller is excluded? Shouldn't be necessary?

Reply:
Nothing much in the controller in terms of business logic and hence excluded.